### ISSUE TITLE
Delete call appears to be missing some parameters

### ISSUE THREAD

**Issue by Garethp** [mooyoul/aws-cdk-ses-domain-identity#15]
For some reason when using this construct I found myself with my stack in a `UPDATE_ROLLBACK_FAILED` state with the `DomainIdentityIdentityRequestorResource3CDC595E` having a status of `UPDATE_FAILED` with message

```
Received response status [FAILED] from custom resource. 
Message returned: 
Invalid request: Expected exactly one of [AliasTarget, all of [TTL, and ResourceRecords], or TrafficPolicyInstanceId], but found none in Change with [Action=DELETE, Name=[random-name].[domain], Type=CNAME, SetIdentifier=null] (RequestId: d6163589-02ac-4729-9d97-895553e364c6)
```

It looks like maybe the API call has changed and [`Record.action`](https://github.com/mooyoul/aws-cdk-ses-domain-identity/blob/master/lambda-packages/dns-validated-domain-identity-handler/src/record.ts#L21) needs to be updated?

**Comment by mooyoul** [comment]
Yes, You're right. Need to fix that. Please see https://github.com/mooyoul/aws-cdk-ses-domain-identity/issues/6#issuecomment-819211344 also. 

I've been super busy recently and still busy 😭 it would be really appreciated if you can make a changes and send me a PR 🙇

**Comment by Garethp** [comment]
Yup. I'm just experimenting with the change locally before I make a PR :)

**Comment by mooyoul** [comment]
This issue was fixed by your PR and released as v1.0.5. Thanks again for your contribution <3 Stay safe be well!
